applications/serial_lte_modem/doc/slm_testing.rst

```diff
@@ -1620,7 +1620,7 @@ Complete the following steps to test the functionality provided by the :ref:`SLM
 
          **AT#XFTP="status"**
          215 UNIX Type: L8
-         211 https://www.pureftpd.org/
+         211 https:\ //www.pureftpd.org/
```

Comment:
What are we escaping here?

Reply:
Like this, the link is displayed as plain text and not rendered as clickable link. And since this here is the command output, there shouldn't be a link to click on.

And all rendered links are checked by the link checker and return warnings if they are broken or (like in the case) redirect to another page, that's why I got rid of links that aren't meant to be links.

Reply:
Ah.. makes sense, Tnx!